lib/master.js

```diff
@@ -24,10 +24,10 @@ var default_options = {
 	connect_timeout:     3  * 1000, // 3s
 	ping_timeout:        10 * 1000, // 10s
 	monitor_interval:    1  * 1000, // 1s
-	totalmaxheap:        512 * 1024 * 1024, // 512 MB
+	total_maxheap_mb:    512, // 512 MB
```

Comment:
if you ask me, I would prefer to keep stuff in bytes and do `512 * ONE_MB` instead.